<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Search by name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search by city</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search by address</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search by building date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search by capacity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search by working hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search by website address</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout_the_Developer"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout_the_Developer">
   <property name="text">
    <string>About the Developer</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
